src/runner.ts

```diff
@@ -99,6 +105,7 @@ export class Runner {
   // TODO(usergenic): This constructor is getting long.  Break up some of
   // these stanzas into supporting methods.
   constructor(options: RunnerOptions) {
+    this._color = options.color === 'on';
```

Comment:
Super minor: the description of the argument suggests that the color is on unless the option is given as `'off'`. Consider:

```typescript
this._color = options.color !== 'off';
```